workflow/common/scripts/setup.py

```diff
@@ -60,13 +61,13 @@
 g123_calibration_files = g123_calibration_dir / "{cohort}.yaml"
 
 # Paths to site workflow results.
-site_results_dir = results_dir / "site"
+site_results_dir = analysis_dir / "site"
 os.makedirs(site_results_dir, exist_ok=True)
 jb_source_dir = site_results_dir / "docs"
 jb_build_dir = jb_source_dir / "_build"
 
 # Setup access to malariagen data.
-malariagen_data_cache_path = results_dir / "malariagen_data_cache"
+malariagen_data_cache_path = output_dir / "malariagen_data_cache"
```

Comment:
Move the cache up one directory level, so cache results can be shared between runs.